react/features/base/util/detectDevices.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Returns true if user agent is run on Android.
+ *
+ * @returns {boolean}
+ */
+export function detectAndroid() {
+    return Boolean(navigator.userAgent.match(/Android/i));
+}
+
+/**
+ * Returns true if user agent is run on iOS.
+ *
+ * @returns {boolean}
+ */
+export function detectIOS() {
+    if (navigator.userAgent.match(/iPhone/i)
+        || navigator.userAgent.match(/iPad/i)
+        || navigator.userAgent.match(/iPod/i)) {
+        return true;
+    }
+
+    return false;
+}
```

Comment:
That's too complicated. It can be improved by:

1. Returning the condition itself.

2. Matching against a single regular expression.